### DISCUSSION TITLE
compile error

### DISCUSSION THREAD

**Discussion by MachineKe** [tailwindlabs/tailwindcss#12740] [Help]
Failed to compile
./app/globals.css.webpack[javascript/auto]!=!./node_modules/next/dist/build/webpack/loaders/css-loader/src/index.js??ruleSet[1].rules[11].oneOf[12].use[2]!./node_modules/next/dist/build/webpack/loaders/postcss-loader/src/index.js??ruleSet[1].rules[11].oneOf[12].use[3]!./app/globals.css
SyntaxError: Unterminated string constant (6:5)
This error occurred during the build process and can only be dismissed by fixing the error.
![image](https://github.com/tailwindlabs/tailwindcss/assets/110995007/e823bbc7-884f-46b1-a4ea-d50201e78898)

**Comment by wongjn** [comment]
Please consider providing a project that reproduces the unexpected behavior.

**Reply by MachineKe** [reply]
its a newly created next project that wont run

**Reply by wongjn** [reply]
OK, please consider providing the project and we can help troubleshoot for you.

**Comment by alex-krasikau** [comment]
Hello @MachineKe!

First, I recommend removing the project and initializing a new one. Please verify if the completely fresh project builds correctly.

If the build fails on the fresh project, then share the package.json, so we can figure out the versions of the dependencies that caused the failure. If the build works fine, it's either a random bug in the compiler or some changes that you made on top of the fresh setup.

Please inform me of the results.
